Common/include/CConfig.hpp

```diff
@@ -9617,10 +9620,73 @@ class CConfig {
    */
   unsigned short GetKind_Grad_Linear_Solver_Prec(void) const { return Kind_Grad_Linear_Solver_Prec; }
 
-    /*!
+  /*!
    * \brief Get max number of iterations of the for the gradient smoothing.
    * \return Max number of iterations of the linear solver for the gradient smoothing.
    */
   unsigned long GetGrad_Linear_Solver_Iter(void) const { return Grad_Linear_Solver_Iter; }
 
+  /*!
+   * \brief Get parsed SST option data structure.
+   * \return SST option data structure.
+   */
+  SST_ParsedOptions GetSSTParsedOptions(void) const {return sstParsedOptions; }
+
+  /*!
+   * \brief function to parse SST options.
+   * \param[in] SST_Options - Selected SST option from config.
+   * \param[in] nSST_Options - Number of options selected.
+   */
+  SST_ParsedOptions ParseSSTOptions(const SST_OPTIONS* SST_Options, unsigned short nSST_Options){
+    SST_ParsedOptions SSTParsedOptions;
+    const auto sst_options_end = SST_Options + nSST_Options;
+
+    const bool sst_1994  = std::find(SST_Options, sst_options_end, SST_OPTIONS::V1994) != sst_options_end;
+    const bool sst_2003 = std::find(SST_Options, sst_options_end, SST_OPTIONS::V2003) != sst_options_end;
+
+    /* when V2003 is selected, we automatically select sst_m as well */
+    const bool sst_m     = std::find(SST_Options, sst_options_end, SST_OPTIONS::MODIFIED) != sst_options_end || sst_2003;
+
+    const bool sst_sust  = std::find(SST_Options, sst_options_end, SST_OPTIONS::SUST) != sst_options_end || Kind_Turb_Model==TURB_MODEL::SST_SUST;
+    const bool sst_v     = std::find(SST_Options, sst_options_end, SST_OPTIONS::VORTICITY) != sst_options_end;
+    const bool sst_kl    = std::find(SST_Options, sst_options_end, SST_OPTIONS::KL) != sst_options_end;
+    const bool sst_uq    = (std::find(SST_Options, sst_options_end, SST_OPTIONS::UNCERTAINTY) != sst_options_end || using_uq);
```

Comment:
If you want to support the old way of specifying things, I would modify the SST_Options list upstream.